DependencyInjection/SonataAdminExtension.php

```diff
@@ -71,14 +71,8 @@ public function load(array $configs, ContainerBuilder $container)
             );
         }
 
-        // TODO: Go back on xml configuration when bumping requirements to SF 2.6+
```

Comment:
You need to add this to the xml declaration not on the PHP, like the comment says